.github/workflows/ci.yaml

```diff
@@ -100,131 +104,135 @@ jobs:
-        cp deps/reopt-vcg/deps/cvc4-2020-09-16-x86_64-linux-opt dist/reopt-vcg/bin/cvc4
-        cd dist
-        tar cvfz reopt-vcg.tgz reopt-vcg
-    - name: Make reopt-vcg
-      uses: actions/upload-artifact@v2
-      with:
-        name: reopt-vcg
-        path: dist/reopt-vcg.tgz
-  integration-tests:
-    needs: [build, reopt-vcg]
-    runs-on: ubuntu-20.04
-    steps:
-    - name: Install dependencies
-      run: sudo apt-get update && sudo apt-get install -y gdb llvm-8
-    - name: Checkout
-      uses: actions/checkout@v2
-    - name: Download reopt artifact
-      uses: actions/download-artifact@v2
-      with:
-        name: reopt
-        path: dist
-    - name: Download reopt-vcg artifact
-      uses: actions/download-artifact@v2
-      with:
-        name: reopt-vcg
-        path: dist
-    - name: Unpack and add executables to PATH
-      run: |
-        tar xvfz reopt.tgz
-        tar xvfz reopt-vcg.tgz
-        echo "$PWD/reopt/bin" >> $GITHUB_PATH
-        echo "$PWD/reopt-vcg/bin" >> $GITHUB_PATH
-      working-directory: dist
-    - name: Test nweb
-      run: ./integration_test.sh
-      working-directory: vcg-test-programs/nweb
-  upload:
-    needs: [build, vscode-extension, reopt-vcg]
-    runs-on: ubuntu-20.04
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-      with:
-        submodules: true
-    - name: Download reopt artifact
-      uses: actions/download-artifact@v2
-      with:
-        name: reopt
-        path: dist
-    - name: Download reopt-vcg artifact
-      uses: actions/download-artifact@v2
-      with:
-        name: reopt-vcg
-        path: dist
-    - name: Copy files
-      run: |
-        cd containers/eval
-        tar xvfz ../../dist/reopt.tgz
-        tar xvfz ../../dist/reopt-vcg.tgz
-    - name: Download extension artifact
-      uses: actions/download-artifact@v2
-      with:
-        name: vscode-extension
-        path: containers/eval
-    - name: Build docker image
-      run: |
-        docker build -t galoisbinaryanalysis/reopt:head -t galoisbinaryanalysis/reopt:latest containers/eval
-    - name: Push to Docker hub
-      if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
-      run: |
-        echo '${{ secrets.DOCKER_PASSWORD }}' | docker login -u galoisbinaryanalysisbot --password-stdin
-        docker push galoisbinaryanalysis/reopt --all-tags
-    - name: Build docker image with examples
-      run: |
-        docker build -t galoisbinaryanalysis/reopt:latest-with-examples -f containers/eval-with-examples/Dockerfile .
-    - name: Push to Docker hub
-      if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
-      run: |
-        echo '${{ secrets.DOCKER_PASSWORD }}' | docker login -u galoisbinaryanalysisbot --password-stdin
-        docker push galoisbinaryanalysis/reopt:latest-with-examples
+
+  # # This builds the VSCode extension
+  # vscode-extension:
```

Comment:
FYI, it may be easier to disable jobs just by adding an `if: false` clause to them.